modules/Microsoft.Compute/sshPublicKeys/.test/.scripts/New-SshPubKey.ps1

```diff
@@ -0,0 +1,9 @@
+﻿param(
```

Comment:
Hey @shawntmeyer you could not have known, but please copy & use this [script](https://github.com/Azure/ResourceModules/blob/main/modules/Microsoft.Compute/virtualMachines/.test/.scripts/New-SSHKey.ps1).

To make matters easier in the future, we also have an [issue](#2619) open to centralize the used test utilities.